cft-templates/s3.yml

```
@@ -16,10 +16,17 @@ Parameters:
     AllowedValues:
       - aws:kms
       - AES256
-
+  AccessLoggingBucketName:
+    Type: String
+    Description: "Pass the bucket name where Access Logs for this S3 bucket should be sent"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)